test/ApiTestCase.php

```diff
@@ -768,6 +768,20 @@ private function createArticlePoAJson(string $id, bool $isSnippet = false, bool
                     ],
                 ],
             ],
+            'relatedArticles' => $isSnippet === false ? [
```

Comment:
It creates an infinite loop when calling `createArticlePoAJson()` and `createArticleVoRJson()` so I had to add that additional check